packages/twenty-front/src/modules/object-record/record-table/record-table-row/components/RecordTableDraggableTr.tsx

```diff
@@ -0,0 +1,64 @@
+import { useTheme } from '@emotion/react';
+import {
+  Draggable,
+  DraggableId,
+  DraggableProvided,
+  DraggableStateSnapshot,
+} from '@hello-pangea/dnd';
+import { forwardRef, ReactNode } from 'react';
+
+import { RecordTableTr } from '@/object-record/record-table/record-table-row/components/RecordTableTr';
+import { mergeRefs } from '~/utils/mergeRefs';
```

Comment:
not sure to understand why we need useRef and mergeRefs here

Reply:
Because we need to manipulate the forwarded ref in the wrapper